src/python/T0/RunConfig/Tier0Config.py

```diff
@@ -789,6 +789,15 @@ def addRepackConfig(config, streamName, **options):
     else:
         streamConfig.Repack.BlockCloseDelay = options.get("blockCloseDelay", 24 * 3600)
 
+    if hasattr(streamConfig.Repack, "SetStreamMemory"):
```

Comment:
I would change the name of this attribute, as this name suggest an action (reads like a method's name). I would use something like `"MaxMemory"` instead.